interview-questions.md

```diff
@@ -8,44 +8,60 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What does calling super do in a Ruby class?
 
-Your answer:
-
-Researched answer:
+Your answer: Calling super in a Ruby class is an inheritance property. It allows Child classes to 'borrow' parameters/key instance variable from their Parent/Super class. This allows us to build subclasses that share similar elements and properties with super classes. By borrwing keys, methods can also be passed down. Heres an example of the syntax:
+
+class Animal 
+    attr_accessor :alive, :age
+    def initialize(age = 0)
+        @alive = true
+        @age = 0
+    end
+end
+
+class Salmon < Fish
+    attr_accessor :alive, :age, :coldblooded, :species
+    def initialize(species, age = 0)
+        super(age = 0)  <----------------------- here 'super' calls to 'Animal' to borrow its '@age' key instance variable.
```

Comment:
Love that you are using an example in your interview questions! However, the `age = 0` within the `super()` and `initialize()` is unnecessary, as having access to the instance variables that are already in the Parent class is the point of using `super()`.